### PR TITLE
Add Quesma to list of the partners

### PR DESCRIPTION
**Pull request by jakozaur**
Elastic / OpenSearch static is popular in observability (e.g. trip.com uses Kibana). With Quesma, it is possible to use it natively with Clickhouse. Added link.

### CODE REVIEW

**Review by pjhampton — CHANGES_REQUESTED** [review]
https://quesma.com/quesma-for-elk has no mention of ClickHouse. The only references I could see to ClickHouse are in an image on the home page and the downloadable PDF referenced as `Elastic / OpenSearch Stacks Interoperability`.

I would be beneficial to get some more ClickHouse specific detail on this integration.

**Comment by jakozaur** [comment]
@pjhampton, we updated the website as you suggested. Please check again.

We also had a previous conversation with @Ryado from ClickHouse about Quesma.

File: docs/en/integrations/index.mdx [hunk 1]
```diff
@@ -187,6 +188,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Popsink|<PopsinkSVG style={{width: '3rem', 'height': '3rem' }} />|Data integration|Build real-time Change Data Capture (CDC) pipelines to ClickHouse.|[Documentation](https://docs.popsink.com/connectors/target/clickhouse/)|
 |PeerDB|<img src={require('./images/logos/logo_peerdb.jpg').default} class="image" alt="PeerDB logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data integration|PeerDB provides a fast, simple and cost-effective way to replicate data from Postgres to Data Warehouses, Queues and Storage.|[Documentation](https://docs.peerdb.io/connect/clickhouse)|
 |Prequel|<PrequelSVG style={{width: '3rem'}} />|Data sharing|Connect your ClickHouse instance to Prequel to share data to or sync data from your users and partners.|[Documentation](https://docs.prequel.co/docs/sources-clickhouse-generic)|
+|Quesma|<QuesmaSVG style={{width: '3rem'}} />|Data integration|Use Elasticsearch / OpenSearch stack and API. E.g. Kibana and OpenSearch Dashboards with data in ClickHouse.|[Website](https://quesma.com/quesma-for-elk)|
```
Comment (pjhampton) [inline docs/en/integrations/index.mdx:191]: How about:

```suggestion
|Quesma|<QuesmaSVG style={{width: '3rem'}} />|Data integration|Use Kibana and OpenSearch Dashboards with data in ClickHouse.|[Website](https://quesma.com/quesma-for-elk)|
```

**Comment by jakozaur** [comment]
@pjhampton applied fixes and styled our logo for the `light` and `dark` theme.
Screenshot 2024-06-18 at 20 36 10
Screenshot 2024-06-18 at 20 36 04

**Review by pjhampton — APPROVED** [review]
Rock on! Look forward to seeing more from Quesma in the future!

**Pull request merged**